app/js/components/manageApps/ManageApps.jsx

```diff
@@ -362,8 +362,11 @@ export default class ManageApps extends React.Component {
     });
   }
 
-  handleUserClick(name) {
-    toastr.info(`Sorry, there is no open web app for ${name}`);
+  handleUserClick(appType, name) {
+    appType === "module"?
+      toastr.info(`Sorry, there is no open web app for ${name}`)
+      :
+      toastr.info(`${name} is not installed`);
```

Comment:
This would me more helpful if it also tells the user what they actually need to do.

Reply:
I have mad it such that the user is informed to install the OWA to access the link to it's app. As for the message on modules, the message that is present clearly informs the user that the OWA does not exist.